.github/workflows/spell_check.yml

```diff
@@ -0,0 +1,28 @@
+name: Spell check CI
```

Comment:
Looking forward to the completion of this PR!

Reply:
@eniac871 FYI

Reply:
I'm getting a headache on doing this